Mathlib/Algebra/Field/Basic.lean

```diff
@@ -47,10 +48,14 @@ theorem div_add_one (h : b ≠ 0) : a / b + 1 = (a + b) / b :=
   (div_add_same h).symm
 #align div_add_one div_add_one
 
+theorem inv_mul_add_mul_inv_eq_inv_add_inv (ha : a ≠ 0) (hb : b ≠ 0) :
+    a⁻¹ * (a + b) * b⁻¹ = a⁻¹ + b⁻¹ :=
+  let _ := invertibleOfNonzero ha; let _ := invertibleOfNonzero hb;
```

Comment:
```suggestion
  let _ := invertibleOfNonzero ha; let _ := invertibleOfNonzero hb
```

Reply:
I changed my lemma names and now this all fits on one line.